.github/workflows/adiciona_issues_project_mentoriaiac.yaml

```
@@ -0,0 +1,12 @@
+name: "Adiciona Issues em Projects GitHub"
+  
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  add-issue:
+    uses: "mentoriaiac/cicd_centralizado/.github/workflows/add_issues_projects.yaml@v1"
+    with:
+      project-url: https://github.com/orgs/mentoriaiac/projects/6
```

```suggestion
      project_url: https://github.com/orgs/mentoriaiac/projects/6
```

A sua variável usa `_` ao invés de `-`.